subsys/caf/modules/power_manager.c

```diff
@@ -76,6 +85,42 @@ static void power_down_counter_abort(void)
 	LOG_DBG("Power down timer aborted");
 }
 
+#if IS_ENABLED(CONFIG_CAF_POWER_MANAGER_RUNTIME_TIMEOUT_SET)
```

Comment:
I bet you can safely remove the preprocessor exclusion. The optimizer will remove these functions anyway.
And we will have a free check done by compiler.